src/lib/tabs/tab-group.ts

```diff
@@ -275,4 +275,31 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
     this._tabBodyWrapperHeight = this._tabBodyWrapper.nativeElement.clientHeight;
     this._renderer.setStyle(this._tabBodyWrapper.nativeElement, 'height', '');
   }
+
+  /** Finds the next tab in the stack for the given direction */
+  _findNextTabByIndex(index: number, direction: string): number {
```

Comment:
Similar function elsewhere in library: https://github.com/angular/material2/blob/master/src/cdk/a11y/list-key-manager.ts#L203